new_client/src/actions/authActions.js

```diff
@@ -0,0 +1,100 @@
+import {
+  USER_LOADED,
+  USER_LOADING,
+  AUTH_ERROR,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+  REGISTER_SUCCESS,
+  REGISTER_FAIL
+} from "./types";
+import axios from "axios";
+import { returnErrors } from "./errorActions";
+
+export const loadUser = () => (dispatch, getState) => {
+  dispatch({ type: USER_LOADING });
+
+  axios
+    .get("http://localhost:5000/api/user", tokenConfig(getState))
+    .then(res => res.json())
+    .then(res => dispatch({
+      type: USER_LOADED,
+      payload: res.data
+    }))
+    .catch(err => {
+      dispatch(returnErrors(err.response.data, err.response.status));
+      dispatch({
+        type: AUTH_ERROR
+      })
+    })
+}
+
+export const register = ({ username, email, password, graduationYear }) => (dispatch) => {
+  const config = {
+    headers: {
+      "Content-Type": "application/json"
+    }
+  };
+  const body = JSON.stringify({ username, email, password, graduationYear });
+  console.log(body);
+  console.log(process.env.REACT_APP_API_REGISTER)
```

Comment:
It is definitely okay to log your debugging variables when testing/trying out stuff. But when making a PR, please make sure to remove such logs, unless they are important.